packages/design-system/src/MetaBox.tsx

```diff
@@ -0,0 +1,20 @@
+import { Box, Heading } from '@chakra-ui/core';
+import React from 'react';
+
+export type MetaBoxProps = {
+  title: string;
+  children: React.ReactNode;
+};
+
+export const MetaBox: React.FC<MetaBoxProps> = ({ children, title }) => (
+  <Box>
+    <Box bg="#392373" p={4}>
```

Comment:
Can we define these values in the theme and refer to them by name instead of hardcoding them?

Reply:
sure, thanks for the feedback